packages/dev-env/src/bsky.ts

```diff
@@ -88,6 +88,7 @@ export class TestBsky {
       indexerPartitionIds: [0],
       indexerNamespace: `ns${ns}`,
       indexerSubLockId: uniqueLockId(),
+      pushNotificationEndpoint: 'https://push.bsky.app/api/push',
```

Comment:
this should be some local url that just no-ops - we don't want to push test notifs to our actual notif service

Reply:
I asked Jake about this but he said its fine to do so. Let me know if not.

Reply:
`gorush` has a binary we can ship with `dev-env` but will probably need @Jacob2161 to set that up